Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Pointers/ParabolicPointer.prefab

```diff
@@ -365,6 +365,9 @@ MonoBehaviour:
   lineMaterial: {fileID: 2100000, guid: 11727442de02c1d4b8d37a063c748aec, type: 2}
   roundedEdges: 1
   roundedCaps: 1
+  fadeLineBrightnessOnEnable: 1
```

Comment:
Are these expected?

Reply:
Yes these are just the default values in the MRLineRenderer class. At some point they were added but this prefab was never updated. When executed at runtime, Unity just takes the defaults even though there is no entry in the text file. 

Basically no one has opened and saved the prefab asset since these new line renderer fields were added